example/pubspec.yaml

```diff
@@ -11,6 +11,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+#  advanced_storybook: ^1.0.2
+  widgetbook: ^2.4.1
+  ionicons: ^0.2.1
```

Comment:
Something's not quite right with the icons on Chrome: 
<img width="759" alt="image" src="https://user-images.githubusercontent.com/40931732/184655585-7ec551b7-37d9-4f80-9c4e-1f829bc5c314.png">